File: MessagingPrototype/src/main/scala/people/Employee.scala
package people

import javax.persistence.{GeneratedValue, Id, Entity}

/**
 * Created by justusadam on 19/11/14.
 */
@Entity
protected class Employee extends Person{

  @Id
  @GeneratedValue
  private var _id:Int = _
  private var _name:String = _

  def this(id:Int, name:String) = {
    this()
    _id = id
    _name = name
  }


  def name = _name
  def id = _id

  def getId = _id

  def getName = _name
}

File: ManagerPrototype/src/main/java/fviv/model/Employee.java
package fviv.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

import org.salespointframework.useraccount.UserAccount;

@Entity
public class Employee extends Person{
	@Id
	@GeneratedValue
	private long id;

	public Employee(String lastname, String firstname, String email, String phone){
		super(lastname, firstname, email, phone);
	}
	
	public long getId(){
		return id;
	}
}
